packages/extension-browser/src/_locales/en/messages.json

```diff
@@ -111,6 +127,10 @@
         "message": "None",
         "description": "Label for option to not ignore any resources on the page."
     },
+    "noThanks": {
+        "message": "No Thanks",
```

Comment:
```suggestion
        "message": "No, thanks",
```

Reply:
Whoa, the comma feels weird here 😵(though possibly more technically correct). Personally I prefer the existing treatment of "No Thanks".